libs/tailwindcss-config/src/vega-custom-classes.js

```diff
@@ -40,6 +40,9 @@ const vegaCustomClasses = plugin(function ({ addUtilities }) {
     '.color-scheme-dark': {
       colorScheme: 'dark',
     },
+    '.rtl-dir': {
```

Comment:
Maybe it will be easier to add all css rules (whitespace-nowrap overflow-hidden text-ellipsis text-right) to this class and rename it to something like .truncate-number

Reply:
imho this rtl is so uniq and not covered by tailwind, so let's not break the pattern of avoiding custom classes if it not necessary